tests/installation/ipxe_install.pm

```diff
@@ -87,19 +87,44 @@ sub set_bootscript {
         $initrd .= "/boot/$arch/loader/initrd";
     }
 
+    if (get_var('SUT_NETDEVICE')) {
+        my $interface = get_var('SUT_NETDEVICE');
+        $install .= "?device=$interface ifcfg=$interface=dhcp4 ";
+    }
 
     my $cmdline_extra;
     $cmdline_extra .= " regurl=$regurl " if $regurl;
     $cmdline_extra .= " console=$console " if $console;
 
+    $cmdline_extra .= get_var('EXTRA_PXE_CMDLINE') if get_var('EXTRA_PXE_CMDLINE');
     $cmdline_extra .= " root=/dev/ram0 initrd=initrd textmode=1" if check_var('IPXE_UEFI', '1');
 
     if ($autoyast ne '') {
         $cmdline_extra .= " autoyast=$autoyast ";
     } else {
-        $cmdline_extra .= " sshd=1 vnc=1 VNCPassword=$testapi::password sshpassword=$testapi::password ";    # trigger default VNC installation
+        if (check_var('VIDEOMODE', 'text')) {
+            $cmdline_extra .= 'ssh=1 ';    # trigger ssh-text installation
+        }
+        else {
+            $cmdline_extra .= " vnc=1 VNCPassword=$testapi::password ";    # trigger default VNC installation
+        }
+    }
+    $cmdline_extra .= ' sshd=1 sshpassword=$testapi::password plymouth.enable=0 ';
+
+    # Extra options for virtualization tests with ipmi backend
+    if (get_var('VIRT_AUTOTEST') || get_var('HANA_PERF')) {
+        $cmdline_extra .= " vga=791 video=1024x768 vt.color=0x07";
+        my $serial_dev;
+        if (get_var('IPXE_CONSOLE')) {
```

Comment:
What's the background for differentiating IPXE_CONSOLE and SERIALDEV? It seems both are to speciy the real serial console?

Reply:
yes, IPXE_CONSOLE was introduced by ipxe installation and is being used by kernel team. SERIALDEV is being used by virt team and it is mandatory as it is referenced in host autoyast profiles. So we have to support both of them.

Reply:
usually these value are valid for them:
```
IPXE_CONSOLE=ttyS1,115200
SERIALDEV=ttyS1
```

Reply:
ok, maybe adding a comment to explain them will be useful for reference